<?xml version="1.0"?>
<launch>
   <node name="rona_exploration_node" pkg="rona_exploration" type="rona_exploration_node">
      <param name="map_frame"                type="string" value="map"               />
   </node>
</launch>
